client/src/components/TripElement/TripForm.js

```diff
@@ -43,10 +43,10 @@ export function TripForm({ onSubmit, initialValues }) {
       className='flex flex-col space-y-4 text-center'
     >
       <TextField
-        {...register('destination', { required: true })}
+        {...register('tripLocation', { required: true })}
         label='Destination'
```

Comment:
i dont think it matters, but should we label it as triplocation as well?

Reply:
actually, nvm. i think it makes more sense to display "destination" to users instead of "tripLocation"
<img width="157" alt="Screenshot 2023-07-01 at 2 19 47 PM" src="https://github.com/cpsc455-bugstorm/TravelersTea/assets/60766732/db217571-605e-4965-9bd0-b669fb1895af">